### ISSUE TITLE
v1.0.101 panic: runtime error: index out of range [3] with length 1 (MacOS)

### ISSUE THREAD

**Issue by frankf1957** [ElrondNetwork/elrond-go#1359]
node failed three times this morning with the error:
panic: runtime error: index out of range [3] with length 1

Log files attached.

Beagle:node-0 elrond$ ./node --version
Elrond Node CLI App version v1.0.101-0-g294d8ef8a/go1.13.8/darwin-amd64

[elrond-go-2020-06-04-12-15-33.log.gz](https://github.com/ElrondNetwork/elrond-go/files/4438449/elrond-go-2020-06-04-12-15-33.log.gz)
[elrond-go-2020-06-04-12-21-52.log.gz](https://github.com/ElrondNetwork/elrond-go/files/4438450/elrond-go-2020-06-04-12-21-52.log.gz)

**Comment by frankf1957** [comment]
Follow-up:

The panic crash occurred 10 hours ago.  At that time, I restarted within a few minutes, and the crash occurred a second time.  I restated again, and it occurred a third time.  

I renamed logs/, stats/, and db/ then started the node expecting it to crash again once it had sync'd up to the point of the previous crash.  That did not happen, it is still running. 

The crash occurred around DEBUG[2020-04-06 12:19:06.605], shard = 4 epoch = 64 round = 38591 nonce = 38299.  

It is currently INFO [2020-04-06 22:50:36.419], epoch = 74 round = 44906 nonce = 44274 shard id = 4 .
